### PR TITLE
Feature/add logout button

### PR DESCRIPTION
**Pull request by Gladdy17**

### CODE REVIEW

File: pawsitive/config/routes.rb [hunk 1]
```diff
@@ -11,4 +11,6 @@
 
   # Defines the root path route ("/")
    root "home#index"
+   delete '/sessions', to: 'sessions#destroy'
+
```
Comment (lincoln1883) [inline pawsitive/config/routes.rb]: Hello @Gladdy17,it is not necessary to explicitly provide a route to delete it is already a devise helper function just specify the button_to.

File: pawsitive/app/controllers/users_controller.rb [hunk 2]
```diff
@@ -4,4 +4,9 @@ class UsersController < ApplicationController
   def show
     current_user = User.find_by(id: params[:id])
   end
+
+  def destroy
+    session.clear
+    redirect_to root_path
+  end
```
Comment (lincoln1883) [inline pawsitive/app/controllers/users_controller.rb]: Also this would not be necessary either because devise works behind the seen.

Comment (Gladdy17) [reply pawsitive/app/controllers/users_controller.rb]: Made changes

Comment (lincoln1883) [reply pawsitive/app/controllers/users_controller.rb]: we have a merge conflict.

**Review by lincoln1883 — APPROVED** [review]
Perfect you are good to go.

**Pull request merged**